database/seeds/UserListTableSeeder.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Hedonist\Entities\UserList\UserList;
+
+class UserListTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(UserList::class,3)->create();
```

Comment:
I think we will create more human readable mock data for user lists. You can create 2-3 lists for each user with names like:
* Favorite Lviv cafes
* Favourite Lviv restaurants
* Favourite Lviv cofee